index.html

```diff
@@ -132,10 +132,13 @@
     </div>
 
     <footer>
-        <div>
-            <img class="socialMediaIcon" src="assets/instagram.svg">
-            <img class="socialMediaIcon" src="assets/facebook.svg">
-            <img class="socialMediaIcon" src="assets/linkedin.svg">
+        <div class="logoSoMeWrapper">
+            <img class="footerLogoSoMe" src="assets/logoDarkBlue.svg" />
```

Comment:
There should be alt text for accessibility here.

Consider WebAIM's [guidelines](https://webaim.org/techniques/alttext/).